scripts/lint/spec_lint.py

```
@@ -0,0 +1,74 @@
+
+def b3_hex(data: bytes) -> str:
+    try:
+        import blake3
+    except Exception:
+        raise SystemExit("ERROR: python module 'blake3' not installed; pip install blake3")
+    return blake3.blake3(data).hexdigest()
+
+def validate_schema(report: Path, schema: Path) -> bool:
+    try:
+        import jsonschema
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)